website/content/60-advanced/601-emr-integration/6011-trust/60112-adfs/601200-installwindowsad.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Setup Windows AD"
+chapter = true
+weight = 601200
+autoNav = true
```

Comment:
I see that all ADFS content is under 60112, but files under that folder and weights are showing different number. It needs to reflect folder number like other chapters (e.g. 601120, 601121